.github/workflows/build.yml

```diff
@@ -100,7 +100,6 @@ jobs:
         **/target/surefire-reports/*.xml
 
   docker:
-    needs: [integration]
```

Comment:
I thin you need to rebase with / merge latest master.

Reply:
done